nifi-marklogic-processors/src/test/java/org/apache/nifi/marklogic/processor/ExtensionCallMarkLogicIT.java

```diff
@@ -58,12 +56,12 @@ void simpleGet() {
 
         runner.assertQueueEmpty();
         assertEquals(0, runner.getFlowFilesForRelationship(ExtensionCallMarkLogic.FAILURE).size());
-        assertEquals(1, runner.getFlowFilesForRelationship(ExtensionCallMarkLogic.SUCCESS).size());
+        assertEquals(2, runner.getFlowFilesForRelationship(ExtensionCallMarkLogic.SUCCESS).size());
 
         List<MockFlowFile> results = runner.getFlowFilesForRelationship(ExtensionCallMarkLogic.SUCCESS);
-        assertEquals(1, results.size());
+        assertEquals(2, results.size());
```

Comment:
Why is this 2 now? The `replay` endpoint returns one item, and that one item still gets written to the single `FlowFile`, so I would think there's still just one `FlowFile` going to `SUCCESS`.

Reply:
there are 2 runs at line 44 and 55, first one generates FlowFile sesssion.create() 1st success, and the 2nd run creates another FlowFile from mocks line 49, second SUCCESS. Before 1st run didn't create anything.

Reply:
Oh I see - it's because the runner is run twice.